pkg/querier/blocks_store_queryable.go

```diff
@@ -654,12 +659,14 @@ func (q *blocksStoreQuerier) fetchSeriesFromStores(
 				if h := resp.GetHints(); h != nil {
 					hints := hintspb.SeriesResponseHints{}
 					if err := types.UnmarshalAny(h, &hints); err != nil {
-						return errors.Wrapf(err, "failed to unmarshal series hints from %s", c.RemoteAddress())
+						level.Warn(spanLog).Log("err", errors.Wrapf(err, "failed to unmarshal series hints from %s", c.RemoteAddress()))
```

Comment:
Does make sense to keep the change smaller for now and just ignore the network errors on [line](https://github.com/cortexproject/cortex/pull/4532/files#diff-5840d289b3ce21e878907eb354057ff9b0659d9dfd826112ea8ded6db06b1022R625) and [line](https://github.com/cortexproject/cortex/pull/4532/files#diff-5840d289b3ce21e878907eb354057ff9b0659d9dfd826112ea8ded6db06b1022R819)?